frontend/src/components/public_group/PublicGroupExpensesAndActivity.js

```diff
@@ -0,0 +1,15 @@
+import React from 'react';
+
+import PublicGroupDonations from './PublicGroupDonations';
+import PublicGroupExpenses from './PublicGroupExpenses';
+
+export default props => (
```

Comment:
You can also name the component so that its name appears in stack traces/error messages (same comment applies to all functional components):

```
const PublicGroupExpensesAndActivity = props => (...);
export default PublicGroupExpensesAndActivity;
```